.github/workflows/collection-continuous-integration.yml

```diff
@@ -0,0 +1,258 @@
+name: Collection test suite
+
+on:
+  push:
+  pull_request:
+  schedule:
+  - cron: 3 0 * * *  # Run daily at 0:03 UTC
+
+jobs:
+  build-collection-artifact:
+    name: Build collection
+    runs-on: ${{ matrix.runner-os }}
+    strategy:
+      matrix:
+        runner-os:
+          - ubuntu-latest
```

Comment:
Your sequence indentations are inconsistent across this file. I recommend that you choose one and stick with it. Personally I don't like indented sequences. It is common to document such preference by committing a YAML lint config, e.g. https://github.com/ansible-community/collection_migration/blob/master/.yamllint#L2

Reply:
Yes, it was w/o indentation, I've added it intentionally. For me it's difficult to parse the file w/o indentation.

Reply:
fixed